.env.example

```diff
@@ -16,6 +16,7 @@ CALENDAR_DB_PORT=5432
 CALENDAR_DB_USER=
 COINFI_COFI_STAKING_WALLET_ADDRESS=0xa61B5f29730C7b111C8e0B1e78bD3737d2Bb8684
 COINFI_PRICES_URL=http://localhost:4000/
+COINFI_PRICES_API_URL=
```

Comment:
Hm, can we rename this to `COINFI_POSTGREST_URL` since that's basically what it actually represents?